docs/rp2/quickref.rst

```diff
@@ -59,7 +59,8 @@ Use the :mod:`time <utime>` module::
 Timers
```

Comment:
We need a code example here how to use the timers.
Can look at the ESP32 & ESP8266 quickref's but ensure that the Timer ID is -1 for rp2.